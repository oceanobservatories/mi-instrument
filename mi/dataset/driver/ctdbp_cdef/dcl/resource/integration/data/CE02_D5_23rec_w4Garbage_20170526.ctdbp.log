2017/05/26 00:00:03.513 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 00:00:06.936 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 00:00:20.342 # 11.0523,  3.64537,    6.817, 26 May 2017 00:00:13
2017/05/26 00:02:50.335 # 11.0167,  3.64258,    6.825, 26 May 2017 00:02:43
2017/05/26 00:03:06.822 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 00:15:08.777 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 00:17:52.336 # 11.0067,  3.64170,    6.556, 26 May 2017 00:17:45
2017/05/26 00:18:08.827 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 00:30:08.445 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 00:33:07.821 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 00:45:06.490 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 00:48:05.825 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 01:00:08.312 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 01:02:51.338 # 11.1567,  3.64707,    6.765, 26 May 2017 01:02:44
2017/05/26 01:03:07.818 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 01:15:04.730 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 01:15:08.147 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 01:17:51.338 # 11.0844,  3.64037,    6, 26 May 2017 01:17:44
2017/05/26 01:18:07.822 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 01:30:09.485 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 01:32:52.339 # 11.1679,  3.64341,    6.174, 26 May 2017 01:32:45
2017/05/26 01:33:08.815 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 01:45:07.770 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 01:48:07.820 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 02:00:07.534 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 02:03:06.825 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 02:15:03.925 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 02:15:07.342 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 02:18:06.819 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 02:30:08.523 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 02:32:51.339 # 11.1219,  3.64118,    6.864, 26 May 2017 02:32:44
2017/05/26 02:33:07.822 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 02:45:06.891 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 02:48:06.827 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 03:00:08.659 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 03:03:08.820 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 03:15:08.404 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 03:18:07.824 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 03:30:07.518 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 03:33:06.832 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 03:45:08.027 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 03:48:07.823 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 04:00:04.434 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 04:00:07.851 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 04:02:51.341 # 10.9880,  3.63112,    7.040, 26 May 2017 04:02:44
2017/05/26 04:03:07.826 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 04:15:02.249 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 04:15:05.662 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 04:17:49.341 # 11.0014,  3.63006,    7.018, 26 May 2017 04:17:42
2017/05/26 04:18:05.821 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 04:30:09.329 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 04:32:52.342 # 10.9883,  3.62831,    7.124, 26 May 2017 04:32:45
2017/05/26 04:33:08.824 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 04:45:03.816 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 04:45:07.238 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 04:48:06.831 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 05:00:03.625 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 05:00:07.043 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 05:03:06.823 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 05:15:08.813 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 05:17:52.343 # 10.8552,  3.62009,    6.746, 26 May 2017 05:17:45
2017/05/26 05:18:08.826 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 05:30:10.345 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 05:32:53.343 # 10.8651,  3.62067,    7.344, 26 May 2017 05:32:46
2017/05/26 05:33:09.829 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 05:45:06.465 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 05:48:05.826 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 06:00:02.792 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 06:00:06.212 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 06:03:07.829 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 06:15:07.920 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 06:18:07.833 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 06:30:09.844 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 06:33:10.114 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 06:45:07.431 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 06:48:06.831 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 07:00:07.141 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 07:03:06.835 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 07:15:03.463 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 07:15:06.880 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 07:18:06.829 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 07:30:09.410 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 07:33:08.832 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 07:45:08.365 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 07:48:07.826 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 08:00:02.700 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 08:00:06.117 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 08:03:05.831 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 08:15:07.787 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 08:18:07.904 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 08:30:10.235 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 08:32:53.348 # 10.6828,  3.60766,    6.879, 26 May 2017 08:32:46
2017/05/26 08:33:09.827 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 08:45:07.275 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 08:48:06.848 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 09:00:03.569 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 09:00:06.987 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 09:03:06.826 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 09:15:03.319 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 09:15:06.732 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 09:18:06.830 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 09:30:10.431 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 09:33:09.833 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 09:45:04.844 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 09:45:08.261 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 09:48:07.832 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 10:00:08.023 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 10:03:07.832 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 10:15:02.436 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 10:15:05.856 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 10:18:06.076 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 10:30:10.219 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 10:33:09.829 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 10:45:03.896 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 10:45:07.314 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 10:48:06.836 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 11:00:03.652 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 11:00:07.070 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 11:03:06.848 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 11:15:03.383 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 11:15:06.797 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 11:18:06.833 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 11:30:08.573 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 11:33:08.834 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 11:45:08.233 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 11:48:07.829 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 12:00:04.565 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 12:00:07.982 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 12:02:51.351 # 10.4278,  3.58829,    7.129, 26 May 2017 12:02:44
2017/05/26 12:03:07.833 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 12:15:07.700 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 12:18:07.836 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 12:30:09.580 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 12:33:09.830 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 12:45:03.800 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 12:45:07.218 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 12:48:06.834 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 13:00:08.921 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 13:02:52.352 # -10,  3.58844,    7.164, 26 May 2017 13:02:45
2017/05/26 13:03:08.837 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 13:15:06.712 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 13:18:06.833 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 13:30:08.535 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 13:32:51.352 # 10.3843,  3.58785,    7.104, 26 May 2017 13:32:44
2017/05/26 13:33:07.836 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 13:45:08.257 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 13:47:51.352 # 10.3989,  3.58829,    7.046, 26 May 2017 13:47:44
2017/05/26 13:48:07.841 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 14:00:04.617 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 14:00:08.034 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 14:02:51.352 # 10.3975,  3.58924,    6.785, 26 May 2017 14:02:44
2017/05/26 14:03:07.833 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 14:15:07.739 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 14:18:07.837 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 14:30:09.548 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 14:32:52.354 # 10.4189,  3.59216,    6.970, 26 May 2017 14:32:45
2017/05/26 14:33:08.841 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 14:45:03.815 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 14:45:07.244 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 14:48:06.836 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 15:00:08.948 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 15:03:08.838 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 15:15:03.321 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 15:15:06.740 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 15:18:06.834 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 15:30:08.424 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 15:32:51.354 # 10.4531,  3.59481,    6.924, 26 May 2017 15:32:44
2017/05/26 15:33:07.837 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 15:45:08.180 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 15:48:07.841 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 16:00:02.576 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 16:00:05.994 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 16:03:07.835 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 16:15:07.683 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 16:18:07.839 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 16:30:09.411 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 16:33:09.269 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 16:45:07.211 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 16:48:06.838 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 17:00:03.611 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 17:00:07.028 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 17:03:06.840 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 17:15:06.768 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 17:18:06.855 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 17:30:10.441 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 17:33:09.837 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 17:45:08.248 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 17:48:07.841 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 18:00:08.005 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 18:03:07.835 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 18:15:04.360 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 18:15:07.777 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 18:17:51.359 # 10.5374,  3.60215,    -7, 26 May 2017 18:17:44
2017/05/26 18:18:07.840 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 18:30:09.642 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 18:32:53.358 # 10.4881,  3.60103,    6.692, 26 May 2017 18:32:46
2017/05/26 18:33:09.842 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 18:45:03.857 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 18:45:07.277 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 18:47:50.358 # 10.6724,  3.60847,    6.699, 26 May 2017 18:47:43
2017/05/26 18:48:06.837 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 19:00:08.961 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 19:02:52.359 # 10.7323,  -3,    6.974, 26 May 2017 19:02:45
2017/05/26 19:03:08.841 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 19:15:06.761 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 19:18:06.846 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 19:30:08.366 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 19:33:07.839 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 19:45:04.841 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 19:45:08.258 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 19:48:07.846 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 20:00:08.006 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 20:03:07.846 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 20:15:04.375 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 20:15:07.793 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 20:18:07.844 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 20:30:09.683 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 20:33:09.844 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 20:45:07.270 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 20:48:06.842 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 21:00:09.050 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 21:03:08.842 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 21:15:03.334 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 21:15:06.748 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 21:18:06.848 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 21:30:08.422 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 21:33:07.874 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 21:45:08.220 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 21:48:07.845 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 22:00:07.952 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 22:02:51.360 # 11.3302,  3.64479,    6.530, 26 May 2017 22:02:44
2017/05/26 22:03:07.839 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 22:15:04.275 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 22:15:07.691 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 22:18:07.844 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 22:30:09.424 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 22:32:52.362 # 11.1609,  3.62804,    6.275, 26 May 2017 22:32:45
2017/05/26 22:33:08.886 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 22:45:03.894 [ctdbp:DLOGP3]:€SBE 16plus
2017/05/26 22:45:07.312 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 22:48:06.866 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 23:00:07.070 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 23:03:06.846 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 23:15:08.903 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 23:17:52.363 # 11.4976,  3.64971,    6.765, 26 May 2017 23:17:45
2017/05/26 23:18:08.849 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 23:30:08.418 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 23:33:07.844 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
2017/05/26 23:45:06.634 [ctdbp:DLOGP3]:Instrument Started [Power On]
2017/05/26 23:48:06.849 [ctdbp:DLOGP3]:Instrument Stopped [Power Off]
