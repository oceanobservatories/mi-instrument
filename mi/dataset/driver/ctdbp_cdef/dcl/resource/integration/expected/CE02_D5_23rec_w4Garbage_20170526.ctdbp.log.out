
 subsite  | node  | sensor       | bin        | method      | time                  | deployment | internal_timestamp | port_timestamp        | preferred_timestamp | quality_flag | conductivity | date_time_string | dcl_controller_timestamp | pressure | temp
----------+-------+--------------+------------+-------------+-----------------------+------------+--------------------+-----------------------+---------------------+--------------+--------------+------------------+--------------------------+----------+---------
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704745600 | telemetered | 3704745620.3420000076 |          5 |         3704745613 | 3704745620.3420000076 |      port_timestamp |           ok |      3.64537 |             null |                     null |    6.817 | 11.0523
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704745600 | telemetered | 3704745770.3350000381 |          5 |         3704745763 | 3704745770.3350000381 |      port_timestamp |           ok |      3.64258 |             null |                     null |    6.825 | 11.0167
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704745600 | telemetered | 3704746672.3359999657 |          5 |         3704746665 | 3704746672.3359999657 |      port_timestamp |           ok |       3.6417 |             null |                     null |    6.556 | 11.0067
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704745600 | telemetered | 3704749371.3380002975 |          5 |         3704749364 | 3704749371.3380002975 |      port_timestamp |           ok |      3.64707 |             null |                     null |    6.765 | 11.1567
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704745600 | telemetered | 3704751172.3389997482 |          5 |         3704751165 | 3704751172.3389997482 |      port_timestamp |           ok |      3.64341 |             null |                     null |    6.174 | 11.1679
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704745600 | telemetered | 3704754771.3389997482 |          5 |         3704754764 | 3704754771.3389997482 |      port_timestamp |           ok |      3.64118 |             null |                     null |    6.864 | 11.1219
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704745600 | telemetered | 3704760171.3410000801 |          5 |         3704760164 | 3704760171.3410000801 |      port_timestamp |           ok |      3.63112 |             null |                     null |     7.04 |  10.988
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704745600 | telemetered | 3704761069.3410000801 |          5 |         3704761062 | 3704761069.3410000801 |      port_timestamp |           ok |      3.63006 |             null |                     null |    7.018 | 11.0014
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704745600 | telemetered | 3704761972.3420000076 |          5 |         3704761965 | 3704761972.3420000076 |      port_timestamp |           ok |      3.62831 |             null |                     null |    7.124 | 10.9883
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704745600 | telemetered | 3704764672.3429999352 |          5 |         3704764665 | 3704764672.3429999352 |      port_timestamp |           ok |      3.62009 |             null |                     null |    6.746 | 10.8552
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704745600 | telemetered | 3704765573.3429999352 |          5 |         3704765566 | 3704765573.3429999352 |      port_timestamp |           ok |      3.62067 |             null |                     null |    7.344 | 10.8651
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704745600 | telemetered | 3704776373.3480000496 |          5 |         3704776366 | 3704776373.3480000496 |      port_timestamp |           ok |      3.60766 |             null |                     null |    6.879 | 10.6828
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704745600 | telemetered | 3704788971.3509998322 |          5 |         3704788964 | 3704788971.3509998322 |      port_timestamp |           ok |      3.58829 |             null |                     null |    7.129 | 10.4278
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704745600 | telemetered | 3704794371.3520002365 |          5 |         3704794364 | 3704794371.3520002365 |      port_timestamp |           ok |      3.58785 |             null |                     null |    7.104 | 10.3843
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704745600 | telemetered | 3704795271.3520002365 |          5 |         3704795264 | 3704795271.3520002365 |      port_timestamp |           ok |      3.58829 |             null |                     null |    7.046 | 10.3989
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704745600 | telemetered | 3704796171.3520002365 |          5 |         3704796164 | 3704796171.3520002365 |      port_timestamp |           ok |      3.58924 |             null |                     null |    6.785 | 10.3975
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704745600 | telemetered | 3704797972.3540000916 |          5 |         3704797965 | 3704797972.3540000916 |      port_timestamp |           ok |      3.59216 |             null |                     null |     6.97 | 10.4189
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704745600 | telemetered | 3704801571.3540000916 |          5 |         3704801564 | 3704801571.3540000916 |      port_timestamp |           ok |      3.59481 |             null |                     null |    6.924 | 10.4531
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704745600 | telemetered | 3704812373.3579998016 |          5 |         3704812366 | 3704812373.3579998016 |      port_timestamp |           ok |      3.60103 |             null |                     null |    6.692 | 10.4881
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704745600 | telemetered | 3704813270.3579998016 |          5 |         3704813263 | 3704813270.3579998016 |      port_timestamp |           ok |      3.60847 |             null |                     null |    6.699 | 10.6724
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704745600 | telemetered | 3704824971.3599996567 |          5 |         3704824964 | 3704824971.3599996567 |      port_timestamp |           ok |      3.64479 |             null |                     null |     6.53 | 11.3302
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704745600 | telemetered | 3704826772.3619999886 |          5 |         3704826765 | 3704826772.3619999886 |      port_timestamp |           ok |      3.62804 |             null |                     null |    6.275 | 11.1609
 CE02SHSM | RID27 | 03-CTDBPC000 | 3704745600 | telemetered | 3704829472.3629999161 |          5 |         3704829465 | 3704829472.3629999161 |      port_timestamp |           ok |      3.64971 |             null |                     null |    6.765 | 11.4976

(23 rows)
